clients/spark/core/src/main/scala/io/treeverse/clients/LakeFSContext.scala

```diff
@@ -81,6 +81,7 @@ object LakeFSContext {
   val LAKEFS_CONF_GC_S3_MIN_BACKOFF_SECONDS = "lakefs.gc.s3.min_backoff_secs"
   val LAKEFS_CONF_GC_S3_MAX_BACKOFF_SECONDS = "lakefs.gc.s3.max_backoff_secs"
   val LAKEFS_CONF_GC_INCREMENTAL = "lakefs.gc.incremental"
+  val LAKEFS_CONF_GC_INCREMENTAL_ITERATIONS = "lakefs.gc.incremental.iterations"
```

Comment:
Instead of "iterations" I suggest (thanks to ChatGPT): `num_runs_back`.

Reply:
```suggestion
  val LAKEFS_CONF_GC_INCREMENTAL_NUM_PREVIOUS_RUNS = "lakefs.gc.incremental.use-nth-previous-run"
```

(and you can even pick which of the 2 wordings you prefer here!)